articles/faq/add-Ip-to-existing-paid-plan.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Can I add a Dedicated IP to my existing Paid plan?"
```

Comment:
I tend to think of a Ded IP as a thing - so tend to capitalise - P in paid i can drop